lib/api/core/models/repositories/profileRepository.js

```diff
@@ -8,13 +8,12 @@ module.exports = function (kuzzle) {
     NotFoundError = require('kuzzle-common-objects').Errors.notFoundError,
     Repository = require('./repository');
 
-  var extractRoleIds = function (policies) {
-    return policies.map(policy => {
-      if (typeof policy === 'string') {
-        return policy;
+  var extractRoleIds = policiesOrRoles => {
```

Comment:
Why did we have this kind of name ? Normally we only have to extract roles ids from policies